core/spec/lib/spree/money_spec.rb

```diff
@@ -268,4 +268,32 @@
       end
     end
   end
+
+  describe "<=>" do
```

Comment:
I thought about also adding tests for `<`, `>`, `.between?`, and `.sort` as those also get included with `Comparable` but I didn't want to get into the habit of testing another module. Any thoughts?